core/src/main/java/hudson/slaves/Cloud.java

```diff
@@ -310,8 +317,68 @@ public HttpResponse doDoDelete() throws IOException {
         return new HttpRedirect("..");
     }
 
+
+    @Override
+    public boolean isNameEditable() {
+        return true;
```

Comment:
I must be missing something, because I can't see how renaming is exposed in the GUI for, say, the Docker plugin. I have a Docker cloud created against a core before this PR and the current release of Docker plugin, and I have another Docker cloud created against a core after this PR and https://github.com/jenkinsci/docker-plugin/pull/1016. I can visit `confirm-rename` manually and rename the cloud there, but I can't get the link to show up anywhere in the GUI for either of my two clouds. Please forgive me if I am doing something wrong on my side.

Reply:
The rename page gets exposed via `TransientActionFactoryImpl`, which is hidden in `RenameAction`. I had to get pointed in the right direction as well when I was first looking into this. 
I was able to create an old docker cloud and then when I changed to https://github.com/jenkinsci/docker-plugin/pull/1016, was able to see the old docker cloud name and then rename it though. But to be fair, I didn't really have any parameters filled in my cloud.

Reply:
I still couldn't find where `RenameAction` is visible in the UI on `Cloud` with these changes even after looking on several pages. I must be missing something.